Sources/FCM/FCMAndroidConfig/FCMAndroidNotification.swift

```diff
@@ -1,75 +1,103 @@
+        case color
+        case sound
+        case tag
+        case clickAction = "click_action"
+        case bodyLocalizationKey = "body_loc_key"
+        case bodyLocalizationArgs = "body_loc_args"
+        case titleLocalizationKey = "title_loc_key"
+        case titleLocalizationArgs = "title_loc_args"
+        case channelID = "channel_id"
+        case ticker
+        case sticky
+        case eventTime = "event_time"
+        case localOnly = "local_only"
+        case notificationPriority = "notification_priority"
+        case defaultSound = "default_sound"
+        case defaultVibrateTimings = "default_vibrate_timings"
+        case defaultLightSettings = "default_light_settings"
+        case vibrateTimings = "vibrate_timings"
+        case visibility
+        case notificationCount = "notification_count"
+        case lightSettings = "light_settings"
+        case image
+    }
+    
     /// The notification's title.
     /// If present, it will override FCMNotification.title.
-    public var title: String?
+    public let title: String?
     
     /// The notification's body text.
     /// If present, it will override FCMNotification.body.
-    public var body: String?
+    public let body: String?
     
     /// The notification's icon. Sets the notification icon to myicon for drawable resource myicon.
     /// If you don't send this key in the request,
     /// FCM displays the launcher icon specified in your app manifest.
-    public var icon: String?
+    public let icon: String?
     
     /// The notification's icon color, expressed in #rrggbb format.
-    public var color: String?
+    public let color: String?
     
     /// The sound to play when the device receives the notification.
     /// Supports "default" or the filename of a sound resource bundled in the app. Sound files must reside in /res/raw/.
-    public var sound: String?
+    public let sound: String?
     
     ///Identifier used to replace existing notifications in the notification drawer. If not specified, each request creates a new notification. If specified and a notification with the same tag is already being shown, the new notification replaces the existing one in the notification drawer.
-    public var tag: String?
+    public let tag: String?
     
     /// The action associated with a user click on the notification.
     /// If specified, an activity with a matching intent filter is launched when a user clicks on the notification.
-    public var click_action: String?
+    public let clickAction: String?
     
     /// The key to the body string in the app's string resources
     /// to use to localize the body text to the user's current localization.
     /// See String Resources for more information.
-    public var body_loc_key: String?
+    public let bodyLocalizationKey: String?
     
-    /// Variable string values to be used in place of the format specifiers
+    /// letiable string values to be used in place of the format specifiers
```

Comment:
That's funny 🙂 definitely should be 'Variable' 🙂

Reply:
Haha search and replace!